packages/features/ee/organizations/lib/orgDomains.ts

```diff
@@ -20,7 +20,7 @@ export function getOrgDomain(hostname: string) {
   const currentHostname = appHostnames.find((ahn) => {
     const url = new URL(WEBAPP_URL);
     const testHostname = `${url.hostname}${url.port ? `:${url.port}` : ""}`;
-    return testHostname.endsWith(`.${ahn}`);
+    return testHostname.endsWith(ahn);
```

Comment:
This is unrelated, but I was getting 404 error in organization and users profile.